index.html

```diff
@@ -16,9 +16,9 @@
           <a href="#"><img class="button-mobile" src="img/mobile-button.png" alt="This is a mobile icon"/></a>
```

Comment:
- [ ] You added the hamburger icon as an image here, it should not be an image but an icon so that it is clickable and displays the mobile menu. You can check [Font Awesome](https://fontawesome.com/v5.15/icons/bars?style=solid) for the icon.